ably/types/options.py

```diff
@@ -10,6 +10,7 @@ def __init__(self, client_id=None, log_level=0, tls=True, rest_host=None,
                  queue_messages=False, recover=False, environment=None,
                  http_open_timeout=None, http_request_timeout=None,
                  http_max_retry_count=None, http_max_retry_duration=None,
+                 fallback_hosts_use_default=True, fallback_hosts=[],
```

Comment:
As above, `fallback_hosts_use_default` does not default to `true`.  Also, fallback_hosts should default to `None`, because an empty array indicates no fallback hosts should be used.  See http://docs.ably.io/client-lib-development-guide/features/#RSC15a, which states `If an array of custom fallback hosts are provided in ClientOptions#fallbackHosts, then they will be used instead. If an empty array of fallback hosts is provided, then fallback host functionality is disabled`